qa/L0_config/test.sh

```diff
@@ -73,32 +72,24 @@ for config in ${LIST_OF_CONFIG_FILES[@]}; do
         cat $ANALYZER_LOG
         RET=1
     fi
-    MODEL_ANALYZER_ARGS="$MODEL_ANALYZER_ANALYZE_BASE_ARGS"
-    MODEL_ANALYZER_SUBCOMMAND="analyze"
-    run_analyzer
```

Comment:
Should we still check if the analyze subcommand works in the L0 tests since we haven't entirely deprecated it yet?

Reply:
There's separate tests (see L0_analyze) that still use the analyze subcommand.

Reply:
Gotcha. Thanks!